src/lib.rs

```diff
@@ -1,6 +1,6 @@
 mod tetragon {
     // NOTE: Since this is generated by tonic theres nothing else we can do.
-    #![allow(clippy::large_enum_variant)]
+    #![allow(clippy::all)]
```

Comment:
I still don't see this at the top of generated module @Aditeya

Reply:
I don't understand, do you mean the `tetragon.rs` file that's generated or the lib.rs file?

Reply:
https://github.com/tokio-rs/prost/issues/661#issuecomment-1156606409

Reply:
Hmm .. I guess clippy cli options are overwriting it. Even when i add `#[clippy(allow::all)]`, clippy still fails because of `-Dmissing_docs` passed to thr cli.

Reply:
apprently `missing_docs` isn't under `clippy::*`. I pushed a fix.